docs/20_smart-contracts/10_getting-started/20_accounts/index.md

```diff
@@ -0,0 +1,208 @@
+
+
+![Weights and thresholds](images/weights_and_thresholds.png)
+
+
+## Smart Contracts
+
+A smart contract is just a program that runs on the blockchain. It allows you to add functionality to an account ranging from simple things like a todo application to a fully-fledged RPG game running entirely on the blockchain. 
+
+Every account has the ability to have one single smart contract deployed to it, however, that smart contract can be updated and replaced at will. 
+
+For more information about deploying smart contracts to your accounts, please see our **DUNE Guide**. 
+
+
+## Creating an Account with DUNE
+
+Once you have **DUNE** set up you can start creating accounts on your local development environment with a single command. 
+
+```
+dune --create-account <ACCOUNT_NAME>
+```
+
+If you want to see the information related to the account you just created, you can use the following command. 
+
+```
+dune -- cleos get account
+```
+
+
+## Ownership of Digital Assets
+
+When something is *owned* by an account on a blockchain, it means that a row in the decentralized database says that the digital asset belongs to that account. 
+
+In most cases, only the specific account that is registered as the owner of that digital asset and has the private key that can prove ownership of the account (via a signature) may manipulate that data. 
+
+Keep in mind that a smart contract also co-owns that data with the user, and might be able to manipulate all data that is stored inside of its tables without the explicit consent of the user.
+
+Smart contract developers may also update smart contracts at will, so contracts that have security or financial implications may relinquish their ability to update their smart contracts in order to trade upgradeability for increased user trust. 
+
+
+## Relinquishing ownership of an account
+
+Upgradeability has significant benefits for smart contract development, but isn’t always wanted. 
+
+In order to relinquish a smart contract’s upgradeability, you have two options.
+
+
+### NULL account
+
+You may set the contract account’s owner and active permissions to `eosio.null@active`. This is a `NULL` account that is specifically designed for these purposes. It has no private key or owner. 
+
+This is a good option if you want to relinquish control of this account **forever**.
+
+
+```
+dune -- cleos set account permission <ACCOUNT> active '{"threshold":1,"keys":[],"accounts":[{"permission":{"actor":"eosio.null","permission":"active"},"weight":1},{"permission":{"actor":"<ACCOUNT>","permission":"eosio.code"},"weight":1}],"waits":[]}' owner -p <ACCOUNT>
+dune -- cleos set account permission <ACCOUNT> owner '{"threshold": 1, "keys":[], "accounts":[{"permission":{"actor":"eosio.null","permission":"active"},"weight":1}], "waits":[] }' -p <ACCOUNT>@owner
+```
+
+(note the eosio.code addition for the active permission, which you might need if the account is a smart contract!)

```

Comment:
you might need? or is it a sure thing?

Reply:
If your contract doesn't interact with any other contracts (like moving tokens for instance) then you don't need it.